eng/versioning/version_client.txt

```diff
@@ -35,6 +35,7 @@ com.azure:azure-messaging-eventhubs;5.1.2;5.2.0-beta.3
 com.azure:azure-messaging-eventhubs-checkpointstore-blob;1.1.2;1.2.0-beta.3
 com.azure:azure-messaging-servicebus;7.0.0-beta.5;7.0.0-beta.6
 com.azure:azure-search-documents;11.0.0;11.1.0-beta.2
+com.azure:azure-security-keyvault-administration;4.1.0-beta.1;4.1.0-beta.1
```

Comment:
Any reason why the client is beginning with version 4.1.0 instead of 4.0.0? There's never been a clear champion scenario for how to handle adding a new module into a group later and determining which base version to use. In Storage we being with 12.x.

Reply:
You are right, I guess I just thought to go with the smallest version released from one of the KV libraries. I think we could do `4.0.0` but `1.0.0` feels weird.